wrapanapi/entities/physical_container.py

```diff
@@ -0,0 +1,28 @@
+# coding: utf-8
+"""
+wrapanapi.entities.physical_container
+
+Implements classes and methods related to physical entities such as racks,
+blocks, sleds, chassis or enclosures that contain other physical entities such
+as physical servers.
+"""
+from abc import ABCMeta
+import six
+
+from wrapanapi.entities.base import Entity
+
+
+class PhysicalContainer(six.with_metaclass(ABCMeta, Entity)):
```

Comment:
Would be nice to backport this to all the other entity derived classes for py2+py3 compatibility.

Reply:
👍 
I am only not sure how to test the changes.